pkg/proc/threads.go

```diff
@@ -20,7 +20,7 @@ type Thread interface {
 	Location() (*Location, error)
 	// Breakpoint will return the breakpoint that this thread is stopped at or
 	// nil if the thread is not stopped at any breakpoint.
-	Breakpoint() BreakpointState
+	Breakpoint() *BreakpointState
```

Comment:
We can change it but I liked that it wasn't a pointer because it implied that it can't return nil.

Reply:
Had to change because all methods worked on a pointer receiver and the compiler complained about taking the address of it once things were moved around.